docs/model_inputs_outputs.md

```diff
@@ -0,0 +1,11 @@
+# Model Inputs and Outputs, Shapes and Layouts {#openvino_docs_model_inputs_outputs}
+
+Users interact with a model by passing data to its _inputs_ and retrieving data from its _outputs_ after the inference. A model may have one or multiple inputs and outputs. Normally, in OpenVINO™ toolkit, all inputs and outputs in the converted model are identified in the same way as in the original framework model. 
```

Comment:
If you decided to keep `after the inference`, probably you should add `to its _inputs_ before the inference` or somehow reflow the sentence to make it more fluent. Keep in mind that the user knowns almost nothing about how inference is organized in OpenVINO (and probably everywhere else) at this point. And the purpose of this chapter is not to explain how it works in OV in details, but rather to give some common analogy between model and some function as a black box that have inputs and outputs. Use some appropriate language.

Reply:
Between this and the next paragraphs we have to explain that inputs and outputs are tensors, which are multi-dimension arrays with some numeric data. Probably refer to some basic materials (for example: https://machinelearningmastery.com/introduction-to-tensors-for-machine-learning/#:~:text=with%20sample%20code).-,What%20are%20Tensors%3F,is%20known%20as%20a%20tensor)

Reply:
The first comment is resolved, for the second, additional explanation will be added later